### ISSUE TITLE
add nosmt/smt=? kernel argument like x86,s390

### ISSUE THREAD

**Issue by grooverdan** [linuxppc/linux#200]

**Comment by mpe** [comment]
We've had `smt-enabled` since 2004 :smile_cat: 

https://github.com/mpe/linux-fullhistory/commit/157e7a9617c8d3baa45eda4ff3071c2d0b4ccf9d

Wiring it up to the new parameter names would be good I guess.

**Comment by grooverdan** [comment]
+ add docs :astonished:  for ^ into Documentation/admin-guide/kernel-parameters.txt

**Comment by stewart-ibm** [comment]
Daniel Black <notifications@github.com> writes:
> + add docs :astonished:  for ^ into
> Documentation/admin-guide/kernel-parameters.txt

We should probably also fix the inevitable bugs of using it under OPAL.

-- 
Stewart Smith
OPAL Architect, IBM.

**Comment by mpe** [comment]
Yeah the reason it's undocumented is we don't want to support it, because it is very good at causing bugs.
